db/1.9/modules/ROOT/pages/database-connector-query-single.adoc

```diff
@@ -0,0 +1,204 @@
+== Query a Single Record from a Database Example - Mule 4
+
+The *Single Query* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not apply streaming for any of the returned fields of the record. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, because if your row is too large you could run out of memory.
+Also, if you are using connection pooling you do not have to worry about your application running out of connections, because this operation returns the connection to the pool immediately after the query is executed.
+
+You can invoke the *Single Query* operation mostly in the same way as you do with the *Select* operation, though you always get a single result regardless of the amount of records returned by the actual SQL query.
```

Comment:
Do you need the word "mostly"? 

"invoke the *Single Query* operation mostly in the same way as you do with the *Select* operation, though you always get a single result regardless of the amount of records returned by the actual SQL query."